cincinnati/src/lib.rs

```diff
@@ -19,6 +19,8 @@ extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 
+pub mod common;
```

Comment:
The `cincinnati` crate so far is a client library for graph manipulation (which I was hopefully planning to re-use in the fedora-coreos agent). I think this may not belong here.

Reply:
I wanted to discuss this before merging, thanks for bringing it up. Do we want 4th crate in the repo named "common"?

Reply:
That looks cleaner (maybe a bit overkill for a single function?).

Reply:
I moved it to a new crate anyway. I hope the single function doesn't feel too lonely :upside_down_face: